scripts/common.sh

```diff
@@ -111,7 +111,7 @@ jqIncRawArray() {
     return
   fi
 
-  jq -r ".\"$api\".full.\"$query\"[]" "$conf_file" || {
+  jq -r ".\"$api\".full.\"$query\"[]?" "$conf_file" || {
```

Comment:
I think I'd prefer that we keep this logic the same for both naked and full. So either revert of this, or also apply the same optional logic to the naked as well.